spacy_loggers/mlflow.py

```diff
@@ -71,7 +71,11 @@ def log_step(info: Optional[Dict[str, Any]]):
                 if losses:
                     mlflow.log_metrics({f"loss_{k}": v for k, v in losses.items()})
                 if isinstance(other_scores, dict):
-                    mlflow.log_metrics(util.dict_to_dot(other_scores))
+                    mlflow.log_metrics(
+                        util.dict_to_dot(
```

Comment:
Seeing how the signature of `dict_to_dot` indicates that the input `dict` can have nested `dict`s as values, it might be a better idea to perform the filtering on the result of `dict_to_dot`, which will be a "flattened" `dict`?